roles/cleanup_rh_ruby25/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Remove rh-ruby25 packages
+  package:
+    state: absent
+    name: rh-ruby25*
```

Comment:
You can optimize a bit by adding `when EL7` to it.

Reply:
Done.